lib/router.js

```diff
@@ -1,41 +1,46 @@
 FlowRouter.route('/', {
-    subscriptions: function(params) {
-        console.log("all the subscriptions goes here!");
-    },
-    action: function(params) {
-    	// Session regsitering
-    	Session.set('loggedIn', false);
-    	Session.set('screenName', '');
+  subscriptions: function(params) {
+    console.log("all the subscriptions goes here!");
+  },
+  action: function(params) {
+  	// Session regsitering
+  	Session.set('loggedIn', false);
+  	Session.set('screenName', '');
 
-        FlowLayout.render('layout', { template: "login" });
-    }
-});
-
-FlowRouter.route('/about', {
-    subscriptions: function(params) {
-        console.log("all the subscriptions goes here!");
-    },
-    action: function(params) {
-        FlowLayout.render('layout', { template: "page", page: "about" });
-    }
+    FlowLayout.render('layout', { template: "login" });
+  }
 });
 
 FlowRouter.route('/room', {
-    subscriptions: function(params) {
-        //Subscriptions
-        Meteor.subscribe('People');
-        Meteor.subscribe('Discussion');
-    },
-    action: function(params) {
-        FlowLayout.render('layout', { template: "room" });
-    }
-});
+  subscriptions: function(params) {
+    //Subscriptions
+    Meteor.subscribe('People');
```

Comment:
We need to register subscriptions. See: https://github.com/meteorhacks/flow-router#subscription-management